jax/_src/lax/lax.py

```diff
@@ -1408,21 +1506,28 @@ def lt(x: ArrayLike, y: ArrayLike) -> Array:
   """
   return lt_p.bind(x, y)
 
+@export
 def convert_element_type(operand: ArrayLike,
                          new_dtype: DTypeLike | dtypes.ExtendedDType) -> Array:
   """Elementwise cast.
 
-  Wraps XLA's `ConvertElementType
-  <https://www.tensorflow.org/xla/operation_semantics#convertelementtype>`_
-  operator, which performs an elementwise conversion from one type to another.
+  This function lowers directly to the `stablehlo.convert`_ operation, which
+  performs an elementwise conversion from one type to another.
```

Comment:
I feel like this maybe understates what this function does. We might not want to get into a "weak types" discussion, but perhaps it's worth at least mentioning that this is a no-op if the current dtype of `operand` matches `new_dtype`?

Should we also mention canonicalization?

Reply:
Thanks for the suggestion – I just pushed an update, let me know what you think!

Reply:
LGTM!